INN/umbrella-ipbs
# Homepage our impact

## Changes

This pull request makes the following changes:

- Adds a new `Homepage Impact` widget area to the homepage just above the homepage bottom widget area
- Adds styling for the above new widget area

Desktop:
![Screen Shot 2020-01-28 at 10 46 28 AM](https://user-images.githubusercontent.com/18353636/73279829-aae44d80-41bb-11ea-9b12-d6a6f0ef6cd5.png)

Mobile:
![screencapture-ipbs-test-2020-01-28-10_46_35](https://user-images.githubusercontent.com/18353636/73279828-aa4bb700-41bb-11ea-9209-1d296d437c96.png)

## Why



For #56

## Testing/Questions

Features that this PR affects:

- Homepage impact section


Questions that need to be answered before merging:

- [ ] Is there a better way to place the title here?
Steps to test this PR:

1. Add a text widget to the new `Homepage Impact` section and set the title as `Our Impact`
2. Add 3 more text widgets to the new `Homepage Impact` section and paste this inside of them
```
<a href="http://ipbs.org"><img class="alignnone wp-image-525 size-medium" src="http://ipbs.test/wp-content/uploads/2020/01/running-4782721_1920-336x224.jpg" alt="" width="336" height="224" /></a>
<h4>Impact Title 1</h4>
<p>Brief blurb Et harum quidem rerum facilis est et expedita distinctio. Non numquam eius modi tempora incidunt ut labore.</p>
``` 
(change the image address to one from your environment)
3. Save all and view the homepage. Verify it looks ok on desktop and mobile
